src/components/Theme/tokens.scss

```diff
@@ -20,25 +20,29 @@
 @mixin tokens() {
   --unit: (1/16) * 1em;
 
-  /* Font Variables */
+  /* Font settings */
   --font-family: 'Open Sans', sans-serif;
-  --font-family-title: var(--font-family);
-  --font-family-subtitle: var(--font-family);
-  --font-family-body: var(--font-family);
 
   --font-size-base: calc(16 * var(--unit));
 
-  --font-size-title: calc(30 * var(--unit));
-  --font-size-subtitle: var(--font-size-base);
-  --font-size-body: var(--font-size-base);
-
   --font-size-large: calc(20 * var(--unit));
   --font-size-small: calc(14 * var(--unit));
   --font-size-x-small: calc(11 * var(--unit));
 
-  --font-weight-title: 600;
-  --font-weight-subtitle: 600;
-  --font-weight-body: 500;
+  --osdk-font-family-title: var(--font-family);
+  --osdk-font-size-title: calc(30 * var(--unit));
+  --osdk-font-weight-title: 600;
+  --osdk-color-content-title: rgba(var(--ods-color-content-main));
+
+  --osdk-font-family-subtitle: var(--font-family);
+  --osdk-font-size-subtitle: var(--font-size-base);
+  --osdk-font-weight-subtitle: 600;
+  --osdk-color-content-subtitle: rgba(var(--ods-color-content-main));
+
+  --osdk-font-family-body: var(--font-family);
+  --osdk-font-size-body: var(--font-size-base);
+  --osdk-font-weight-body: 500;
+  --osdk-color-content-body: rgba(var(--ods-color-content-main));
```

Comment:
@josokinas Tried using the Castor helper tokens in this file, but seems like the CSS values don't get applied so it falls back to browser default when there is no customisation overrides. Hence went back to using the Castor CSS variables instead. Works fine when used within `Theme/style.scss` or the individual components' stylesheet's though, but sticking to the CSS variables for now for consistency.

Reply:
That's really odd, it should not be any different...

@rabelloo have any idea why?

Reply:
If its not scoped it might conflict with Scss own functions, e.g.

`this works`
```scss
@use '@onfido/castor';

color: castor.color('content-main');
```

`this doesn't`
```scss
@import '@onfido/castor';

color: color('content-main');
```

Other than that I'd need some more context, ideally see it happen before/after Sass compilation.

Reply:
Basically this was what I tried: 
```
@use '~@onfido/castor';

@mixin tokens() {
  --osdk-color-content-title: castor.color('content-main');
}
```
and on inspecting in browser looks like that's what gets generated also instead of 
```
--osdk-color-content-title: rgba(var(--ods-color-content-main));
```

Reply:
I think this would work:

```scss
@use '~@onfido/castor';

@mixin tokens() {
  --osdk-color-content-title: #{castor.color('content-main')};
}
```

But it is fine as it is, we're using also CSS variables directly when creating themes on Castor.

Reply:
It's probably what Julius said, when creating CSS variables Sass just interprets the whole thing as a string, so you probably need to execute functions inside the interpolation escape. Forgot about that.